packages/react-docs/src/components/layouts/stack.js

```diff
@@ -52,12 +52,12 @@ const StackDocs = ({ data }) => (
 
 StackDocs.propTypes = propTypes;
 
-export const query = graphql`
-  query StackDocsQuery {
-    componentMetadata(displayName: { eq: "Stack" }) {
-      ...ComponentDocs
-    }
-  }
-`;
+// export const query = graphql`
```

Comment:
Delete commented code.

Reply:
These files should actually be deleted. I missed them. I moved them here to make sure I covered the existing examples.